src/Components/Popular-posts-navbar/Popular-posts-navbar.js

```diff
@@ -12,18 +14,27 @@ const PopularNavbar = () => {
 
     return (
         <nav className='PopNavbar'>
+
+            {/* //? Split the 6 Items up into Left and Right - 5 left/ 1 right to help split in CSS */}
             <div className='LeftButtons'>
 
-                <Button className="Button BestBtn"
+{/* //? Realised the BEST button is only when logged into the updated Reddit */}
```

Comment:
You don't have to do any login stuff (at least not this project!) so this can go or stay you can choose if you are displaying it or not and just talk about it in the retro